tests/resources/install-matomo.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+use Piwik\Access;
+use Piwik\Application\Environment;
+use Piwik\Auth\Password;
+use Piwik\Common;
+use Piwik\Container\StaticContainer;
+use Piwik\Date;
+use Piwik\Plugins\UsersManager\UsersManager;
+use Piwik\Plugins\UsersManager\API as UsersManagerAPI;
+use Piwik\Plugins\SitesManager\API as SitesManagerAPI;
+use Piwik\Site;
+use Piwik\Tracker\Cache;
+use Piwik\Config;
+use Piwik\Db;
+use Piwik\DbHelper;
+use Piwik\Option;
+use Piwik\Plugins\LanguagesManager\API as APILanguageManager;
+use Piwik\Updater;
+use Piwik\Plugins\CoreUpdater;
+
+$subdir = $argv[1];
```

Comment:
Maybe we could add a simple check to make sure it's not run through a web request? Accessing `$argv` should probably already make sure of that though. Not sure if there's any way this could be defined through web. Luckily Register Globals isn't a thing anymore :) 
Just thinking of people who deploy Matomo through git and where those files aren't removed.

Could add the check from `Common::isPhpCliMode` just to be safe